app/auth/serializers.py

```diff
@@ -0,0 +1,47 @@
+import typing as tp
+
+from pydantic import root_validator
+
+from models import Account
+from utils.exceptions import SerializerValidationError
+from utils.serializers import BaseSerializer, serializer_data_type
+from utils.data_process import make_password
+
+
+class AccessTokenSerializer(BaseSerializer):
+    """
+    Serializer for access token
+    """
+
+    read_only_fields = ['user_pk']
+
+    username: str
+    password: str
+    user_pk: tp.Optional[int] = None
+
+    @root_validator
+    def validate_all_values(cls, data: serializer_data_type) -> serializer_data_type:
+        """
+        Validate all values
+        """
+
+        accounts = Account.query.filter_by(username=data.get('username')).all()
```

Comment:
by the way you don't need to fetch all the fields of all the accounts, rather check if they exist

Reply:
also I think you expect only one account, so why all. should be rather one or one_or_none

Reply:
done

Reply:
done